spec/jobs/notify_slack_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe Jobs::NotifySlack do
+
+  before do
+    SiteSetting.slack_outbound_webhook_url = "https://hooks.slack.com/services/X/Y/Z"
+    SiteSetting.slack_enabled = true
```

Comment:
We need to reset the site setting otherwise it'll leak to other tests.

I recently added a helper method to make this easier. 

https://github.com/discourse/discourse/blob/master/spec/support/helpers.rb#L5-L16